torchgeo/models/swin.py

```diff
@@ -16,14 +16,40 @@
 __all__ = ["Swin_V2_B_Weights"]
 
 
+# Custom transform for Sentinel-2 multispectral model inputs; Uses 9 bands.
+class MSS2Transform(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x):
+        # Divide the first 3 channels by 255
+        x[:, :3, :, :] = x[:, :3, :, :] / 255.0
+        # Divide the last 6 channels by 8160 and clip to 0-1
+        x[:, -6:, :, :] = torch.clamp(x[:, -6:, :, :] / 8160.0, 0.0, 1.0)
+        return x
+
+
 # https://github.com/allenai/satlas/blob/bcaa968da5395f675d067613e02613a344e81415/satlas/cmd/model/train.py#L42 # noqa: E501
-# All Satlas imagery is uint8 and normalized to the range (0, 1) by dividing by 255
+# Satlas Sentinel-1 and RGB Sentinel-2 and NAIP imagery is uint8 and is normalized to (0, 1) by dividing by 255. # noqa: E501
 _satlas_transforms = AugmentationSequential(
     K.CenterCrop(256),
     K.Normalize(mean=torch.tensor(0), std=torch.tensor(255)),
     data_keys=["image"],
 )
 
+# Satlas multispectral Sentinel-2 imagery divides first 3 bands by 255 and the following 6 bands by 8160, both clipped to (0, 1). # noqa: E501
+_sentinel2_ms_satlas_transforms = AugmentationSequential(
+    K.CenterCrop(256), MSS2Transform(), data_keys=["image"]
+)
+
+# Satlas Landsat imagery is 16-bit, normalized by clipping some pixel N with (N-4000)/16320 to (0, 1). # noqa: E501
+_landsat_satlas_transforms = AugmentationSequential(
+    K.CenterCrop(256),
+    K.Normalize(mean=torch.tensor(4000), std=torch.tensor(1)),
+    K.Normalize(mean=torch.tensor(0), std=torch.tensor(16320)),
```

Comment:
You can implement the previous clipping by adding the following to the end of the AugmentationSequential for example:

```python
from kornia.contrib import Lambda

Lambda(lambda x: torch.clamp(x, min=0.0, max=1.0))
```